### PR TITLE
Add note styling

### PR DESCRIPTION
**Pull request by anthonydillon**
The note styling for cloud theme is to be 16px and have no underlines on links.

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA:
- Run `gulp build`
- Check the demo has the note element

**Comment by barrymcgee** [comment]
@anthonydillon When I add a link to the note element in the demo - it has an underline?

https://www.dropbox.com/s/n88aidp08a57dd0/Screenshot%202016-08-10%2010.05.31.png?dl=0

File: scss/modules/_typography.scss [hunk 1]
```diff
@@ -22,4 +22,9 @@
     border-radius: 2px;
     color: $mid-grey;
   }
+
+  .note {
+    font-size: 1em;
+    text-decoration: none;
```
Comment (barrymcgee) [inline scss/modules/_typography.scss]: Did you mean to nest an `a { ... }` here?

**Comment by anthonydillon** [comment]
Thanks @barrymcgee, I believe the underlined link is expected behaviour when the link it contained in the note but when the note is a link, for example:
```html
<a href="" class="note">This is a note</a>
```
It is not to be underlined.

**Comment by anthonydillon** [comment]
In fact, on second thoughts I dont like the manipulation of link styling here. Removing the `text-decoration` from this PR.

**Comment by anthonydillon** [comment]
@barrymcgee this better?

**Comment by barrymcgee** [comment]
@anthonydillon Yes, much clearer. 👍

**Pull request merged**